submissions/Roka20012/js-dom/interactive.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const nav = document.getElementById("navbar");
+const text = document.getElementById("text");
+const burger = document.getElementsByClassName("burger2")[0];
+/**
+ * закриває меню якщо ми натискаємо на пункт меню 
+ * коли ширина екрана менша 900px
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function hideBurgerMenu(target) {
+    let computedStyle = getComputedStyle(burger);
+
+    if (computedStyle.left == "15px") {
+        nav.classList.toggle("menu-open");
+        burger.classList.toggle("open");
+    }
+}
+
+/**
+ * додає колір для вибраного елемента
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function setColorForElement(target) {
+    let a = Array.from(nav.children);
+
+    a.forEach((el) => {
```

Comment:
[You can use destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) instead of Array.from. it will be [...nav.children].forEach